README.md

```diff
@@ -207,6 +214,12 @@ Typically, it is easier to get to work with `pdflatex` than PSTricks is.
 - German: [Neue TeX FAQ](https://texfragen.de) - A modern and updated LaTeX FAQ in German.
 - [BibTeX Style Examples](http://www.cs.stir.ac.uk/~kjt/software/latex/showbst.html) - Example output of common BibTeX styles (BST files).
 
+## Showcases
+
+- [Showcase of beautiful typography done in TeX & friends](https://tex.stackexchange.com/questions/1319/showcase-of-beautiful-typography-done-in-tex-friends) - A collection of showcases demonstrating the power of LaTeX.
+- [Showcase of beautiful invitations in TeX](https://tex.stackexchange.com/q/281415/9075) - A collection of invitations typeset using LaTeX.
```

Comment:
A personal preference: I prefer `showcase of invitations` over `collection of invitations`.